.github/workflows/Automation Master Legacy Workflow.yml

```diff
@@ -123,9 +123,19 @@ jobs:
         uses: microsoft/setup-msbuild@v2
         
       - name: Enable Skyline GitHub NuGet Registry
```

Comment:
Add a TODO to refactor this with a loop the next time we need to make changes to these steps.

Reply:
ok